### PR TITLE
SDK changes for the remove jobs

### PR DESCRIPTION
**Pull request by vinay3206**

### CODE REVIEW

**Comment by vinay3206** [comment]
Tests are failing because of new snapshots because of the new environment. Should be fine once API is merged to staging and SDK starts pointing to staging.

Job related tests are passing.

File: src/data/customer/Job.js [hunk 1]
```diff
@@ -275,3 +275,26 @@ export const createJobsAsync = async (routeIds, token) => {
...
+ * @param {string} jobIds comma separated string ex: 1234,3455
+ * @param {string} token
+ * @return {object} Promise resolve/reject
+ */
+export const removeJobsAsync = async (jobIds, token) => {
+  try {
+    await axios({
+      method: 'DELETE',
+      // TODO: need to update this endpoint again when API starts accepting multiple job ids
+      url: endpoints.API_V3.JOB.replace('{0}', jobIds),
```
Comment (maartenhemmes) [inline src/data/customer/Job.js]: @vinay3206 will this be any different if the API changes? You would still pass 'jobIds' no?

Comment (vinay3206) [reply src/data/customer/Job.js]: @maartenhemmes  as discussed with nin. It will be probably changed to query string params

Comment (maartenhemmes) [reply src/data/customer/Job.js]: ok, I think they will still fix it within this sprint, so we might as well wait for that? what do you think?

File: src/data/Endpoint.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
+// TODO: update url when api is merged to develop
 const urlPrefix =
-  'https://b2gp6sn1o4.execute-api.ap-southeast-1.amazonaws.com/staging';
-const laravelUrlPrefix = 'https://api-test.carpal.me';
+  'https://b2gp6sn1o4.execute-api.ap-southeast-1.amazonaws.com/pre-test';
```
Comment (maartenhemmes) [inline src/data/Endpoint.js]: we cannot merge it with pre-test still here.

File: src/data/customer/__test__/job.test.js [hunk 3]
```diff
@@ -161,4 +168,40 @@ describe('job create', async () => {
       expect(errorResponse).toEqual(expected);
     }
   });
+
+  it('should delete the jobs', async () => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 30000;
+    let response;
+    try {
+      // this test is passing right now because API only considers one job Id right now although two are passed with request
```
Comment (maartenhemmes) [inline src/data/customer/__test__/job.test.js]: We need to update this test.
Also we need to test removing a job ID that's part of a different route.

File: src/data/customer/__test__/job.test.js [hunk 4]
```diff
@@ -1,4 +1,9 @@
-import { getJobDetailAsync, getJobSummaryAsync, createJobsAsync } from '../Job';
```
Comment (maartenhemmes) [inline src/data/customer/__test__/job.test.js]: We need to test removing a job ID that's part of a different route:

1. The route is owned by customer but it's a different one than the job ID
2. The route is not owned by customer

File: src/data/customer/__test__/job.test.js [hunk 5]
```diff
@@ -47,13 +52,15 @@ describe('Show job', async () => {
   });
 });
 
-describe('job create', async () => {
+describe('job create and delete test', async () => {
```
Comment (maartenhemmes) [inline src/data/customer/__test__/job.test.js]: Test is currently failing: https://travis-ci.org/CarPalFleet/JavaScript-SDK/builds/423733800?utm_source=github_status&utm_medium=notification (don't worry about the snapshots)

File: package.json [hunk 6]
```diff
@@ -67,6 +67,7 @@
     "lodash.isnumber": "^3.0.3",
     "lodash.isobject": "^3.0.2",
     "lodash.toarray": "^4.4.0",
+    "npm": "^6.4.1",
```
Comment (maartenhemmes) [inline package.json]: @Messhias why was this added?
